resources/test-runner.mjs

```diff
@@ -18,6 +18,30 @@ export class TestRunner {
         this.#frame = frame;
     }
 
+    get frame() {
+        return this.#frame;
```

Comment:
since i'm extending and overriding the runTest function, I had to create getters for the private variables.
just more boilerplate..

Reply:
If there are now external consumers for these, should we just make them public?  cc @rniwa

Reply:
I'm only using getters at the moment, so more boilerplate, but maybe more "secure" than just making them public? 🤷

Reply:
+1 to making the public and lower overhead.